cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -844,4 +848,16 @@ export default {
       Section({ id: 'acc02' }).find(KeyValue('Resource title')).has({ value: instanceData.title }),
     ]);
   },
+  clickNewOrder() {
```

Comment:
already exists in 
https://github.com/folio-org/stripes-testing/pull/2504
InventoryInstance.newOrder();